### DISCUSSION TITLE
lens that require tokens

### DISCUSSION THREAD

**Discussion by joesan210** [ptrumpis/snap-camera-server#85] [Q&A] [answered]
How do i got about unlocking the filters that request 100 tokens? any ideas?

Thanks,
Joe

**Comment by ptrumpis** [comment]
I'm not a snap chat user, can you explain what you mean by 100 tokens?
You are not talking about **`lens.snapchat.com`** are you?

**Comment by joesan210** [comment]
No this is about the snap cam for pc.  I set up the server and it works but
inside the app when using a filter there are some that are locked and say to use tokens.

I have attached a pic. How do you unlock those.  Its on the potato filter.

![Screenshot 2023-11-29 202049](https://github.com/ptrumpis/snap-camera-server/assets/78339481/8abec1fb-2050-4ea2-bcec-90b76ec7fc61)

h

**Reply by ptrumpis** [reply, marked as answer]
This seems to be a mobile app feature only.
You can copy and paste any lens share link into the snap camera search box.

- If the lens has a public share link it should work.
- If there is no share link it won't work.

Or in other words, if the lens is available at https://www.snapchat.com/lens it will be available in snap camera.
